Anviz.SDK/Anviz.SDK.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>

```

Comment:
This will break the build and release of the package. We intentionally target netstandard so it can be used in any version of .net framework

Reply:
This is an oversight on my side. I made this initial change because I did not want the user to have to install another framework. This was a moot concern, but I neglected to remove it before checking it in.

Reply:
No problem :) can you push a commit to fix it?

Reply:
Done. Hope you can see the commit. I usually work in BitBucket, so I'm not very familiar with GitHub.